.circleci/config.yml

```diff
@@ -225,7 +225,7 @@ jobs:
       - docker-pull-build-image
       - run: mkdir test-results
       - docker-run:
-          cmd: make integration-test TEST_V=1 SSH_KEY_PATH="/root/.ssh/${SSH_KEY_NAME}"
+          cmd: TEST_V=1 SSH_GIT_COMMAND="ssh -i /root/.ssh/${SSH_KEY_NAME}" make integration-test
```

Comment:
Where does `SSH_GIT_COMMAND` get used? Googling it, it doesn't appear to be a stand git environment variable, and I can't see it getting read in the code 🤔

Reply:
Yes it's a typo